packages/precommit-hooks/bin/prepare.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
alpine doesn't have bash installed - everything else seems to work on alpine but added flag to allow skipping initialisation explicitly if it becomes an issue. 

We don't currently do this for husky in our standard docker files, meaning it sets up precommit hooks unnecessary.